src/routers/routerV2.ts

```diff
@@ -1,12 +1,23 @@
 import { Router } from 'express';
+import rateLimit from 'express-rate-limit';
 import { getWords, getWord } from '../controllers/words';
 import { getExample, getExamples } from '../controllers/examples';
 import { getNsibidiCharacter, getNsibidiCharacters } from '../controllers/nsibidi';
 import { getTranscription } from '../controllers/speechToText';
+import { getTranslation } from '..//controllers/translation';
```

Comment:
```suggestion
import { getTranslation } from '../controllers/translation';
```

just confirming if we want this as the path

Reply:
honestly surprised this even worked